external/runtime/runtime.depproj

```diff
@@ -11,6 +11,12 @@
   <PropertyGroup Condition="'$(TargetGroup)' == 'uapaot'">
     <ProjectJsonTemplate>$(MSBuildThisFileDirectory)NETNative/project.json.template</ProjectJsonTemplate>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
+    <SkipFilterTargetingPackResolvedNugetPackages>false</SkipFilterTargetingPackResolvedNugetPackages>
   </PropertyGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='uapaot'">
+    <TargetingPackReference Include="System.Private.CoreLib" />
```

Comment:
I see, we now use the `<FileToExclude Include="..">`. I'll revert these changes and put up a PR to move to the new model.